src/psuedocode.md

```diff
@@ -2,5 +2,5 @@ Here are the steps needed to accomplish the task.
 
 1.  I will declare an abject that stores each letter by assigning a the given value called letterValues.
 2. declare a variable that stores the sum of each letter in a word called total.
-3.  using map function split the variable, and using conditinals statment, I will compute each letter a value from the letterValues object. 
```

Comment:
would help to have your steps in normal numerical steps i.e. 1,2,3,4,5,6,7 etc etc